pkg/security/utils/cgroup.go

```diff
@@ -85,15 +123,14 @@ func GetProcContainerID(tgid, pid uint32) (containerutils.ContainerID, error) {
 // GetProcContainerContext returns the container ID which the process belongs to along with its manager. Returns "" if the process does not belong
 // to a container.
 func GetProcContainerContext(tgid, pid uint32) (containerutils.ContainerID, model.CGroupContext, error) {
-	cgroups, err := GetProcControlGroups(tgid, pid)
-	if err != nil || len(cgroups) == 0 {
+	cgroup, err := GetLastProcControlGroups(tgid, pid)
```

Comment:
It's not directly related to your PR, but I wonder if we should add a check and log in case the `cgroup.ID != 0`. It "should" not happen but .. We never knows! WDYT ?

Reply:
definitely, I wan't this PR to be "perf improvement" but no functionality change, but I have another PR coming that parses specifically for the `0::` cgroup